SearchSortedMatrix.java

```diff
@@ -0,0 +1,40 @@
+// Time Complexity : O(log n)
```

Comment:
TC is wrong. What is n?

Reply:
n is number of elements in a given array.

Reply:
Then it is correct